app/assets/v2/css/leaderboard.css

```diff
@@ -1,208 +1,175 @@
-#leaderboard .leaderboard_entry {
-  padding: 10px 0px;
-  color: black;
-  text-decoration: none;
-}
-
-#leaderboard .leaderboard_entry img {
-  max-height: 100px;
-  border-radius: 50%;
-}
-
-#leaderboard .leaderboard_entry a {
-  color: #8b8b8b;
-}
-
-#leaderboard .leaderboard_entry .item-amount {
-  font-size: 1.5em;
-  font-weight: bold;
-  color: #86eec4;
+.gitcoin-background {
+  background: url("/static/v2/images/header-bg.png") #0d023b;
+  background-attachment: fixed;
+  background-position: center 0;
 }
 
-#leaderboard .leaderboard_entry .item-position {
-  color: #8b8b8b;
-}
-
-#leaderboard .table-striped tbody tr:nth-of-type(2n+1) {
-  background-color: #f8f8f8;
-}
-
-#leaderboard .progress_container {
-  padding-top: 2.5em;
-}
-
-#leaderboard .key {
+.leaderboard .key {
   right: 0px;
   display: block;
   position: absolute;
   margin-right: 10px;
   margin-top: 5px;
 }
 
-#leaderboard .podium {
+.leaderboard .podium {
   border-bottom-left-radius: 2px;
   border-bottom-right-radius: 2px;
-  background: #f5136e;
-  /* For browsers that do not support gradients */
-  background: -webkit-gradient(linear, center top, center bottom, from(#f5136e), to(#0f023c));
-  background: -webkit-linear-gradient(#f5136e, #0f023c);
-  background: -moz-linear-gradient(#f5136e, #0f023c);
-  background: -o-linear-gradient(#f5136e, #0f023c);
-  background: -ms-linear-gradient(#f5136e, #0f023c);
-  background: linear-gradient(#f5136e, #0f023c);
+  background: #150069;
+  justify-content: space-around;
 }
 
-#leaderboard .winner {
+.leaderboard .winner {
   position: relative;
   padding: 0px;
-  min-height: 23em;
   text-align: center;
 }
 
-#leaderboard .winner-content {
-  top: 40%;
-  left: 30%;
-  position: absolute;
-  max-width: 45%;
+.interior .body {
+  background-color: #140069;
 }
 
-#leaderboard .winner-content .winner-position {
-  width: 100%;
-  min-height: 100px;
-  background-position: center;
-  background-repeat: no-repeat;
-  background-size: contain;
+.leaderboard .clickable-row {
+  cursor: pointer;
 }
 
-#leaderboard .winner-content .winner-position-first {
-  background-image: url('/static/v2/images/leaderboard-1.png');
+.leaderboard .winner img:hover {
+  opacity: 0.55;
 }
 
-#leaderboard .winner-content .winner-position-second {
-  background-image: url('/static/v2/images/leaderboard-2.png');
+.leaderboard .winner img {
+  max-height: 100px;
+  display: flex;
+  border-radius: 50%;
 }
 
-#leaderboard .winner-content .winner-position-third {
-  background-image: url('/static/v2/images/leaderboard-3.png');
+.leaderboard .winner a {
+  color: #2B75D2;
 }
 
-#leaderboard .winner img {
-  opacity: 0.5;
-  min-height: 23em;
+.leaderboard .winner h6 {
+  font-size: color: #3f78cd;
+  white-space: nowrap;
 }
 
-#leaderboard .winner img:hover {
-  opacity: 0.55;
+.leaderboard .leaderboard_entry img {
+  max-height: 50px;
+  border-radius: 50%;
 }
 
-#leaderboard .winner a {
-  color: white;
+table {
```

Comment:
could we use class `font-weight-bold`  in the table element as opposed to introducing duplicate css ?

Reply:
fixed